src/style.css

```
@@ -0,0 +1,24 @@
+html {
+  margin: 0;
+}
+
+.stars .star {
```

don't increase specificity
```suggestion
.star {
```